2014/11/21 15:43:23 [error] [exception.CException] exception 'CException' with message 'Alias "application.extensions.phpexcel.PHPExcel" is invalid. Make sure it points to an existing PHP file and the file is readable.' in E:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\index.php(20): YiiBase::import('application.ext...', true)
#1 {main}
REQUEST_URI=/EngModule/index.php?r=site/excel
---
2014/11/21 15:45:02 [error] [exception.CException] exception 'CException' with message 'Alias "application.ext.phpexcel.PHPExcel" is invalid. Make sure it points to an existing PHP file and the file is readable.' in E:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\index.php(20): YiiBase::import('application.ext...', true)
#1 {main}
REQUEST_URI=/EngModule/index.php?r=site/excel
---
2014/11/21 15:45:03 [error] [exception.CException] exception 'CException' with message 'Alias "application.ext.phpexcel.PHPExcel" is invalid. Make sure it points to an existing PHP file and the file is readable.' in E:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\index.php(20): YiiBase::import('application.ext...', true)
#1 {main}
REQUEST_URI=/EngModule/index.php?r=site/excel
---
2014/11/21 15:45:04 [error] [exception.CException] exception 'CException' with message 'Alias "application.ext.phpexcel.PHPExcel" is invalid. Make sure it points to an existing PHP file and the file is readable.' in E:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\index.php(20): YiiBase::import('application.ext...', true)
#1 {main}
REQUEST_URI=/EngModule/index.php?r=site/excel
---
2014/11/21 15:45:04 [error] [exception.CException] exception 'CException' with message 'Alias "application.ext.phpexcel.PHPExcel" is invalid. Make sure it points to an existing PHP file and the file is readable.' in E:\xampp\htdocs\yii\framework\YiiBase.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\index.php(20): YiiBase::import('application.ext...', true)
#1 {main}
REQUEST_URI=/EngModule/index.php?r=site/excel
---
2014/11/21 15:54:32 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/21 16:47:27 [error] [exception.CException] exception 'CException' with message 'Property "BuildingPermit.owner_nname" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('owner_nname')
#1 E:\xampp\htdocs\EngModule\protected\views\buildingpermit\permit.php(40): CActiveRecord->__get('owner_nname')
#2 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#3 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', Array, true)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', Array, true)
#5 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(369): CController->renderPartial('permit', Array, true)
#6 [internal function]: BuildingPermitController->actionGeneratepdf('8E1E03060095F46...')
#7 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#8 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#9 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#16 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('generatepdf')
#18 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#19 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/generatepdf&var=8E1E03060095F46B80A1D918AEA77151
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/view&permit_num=72
---
2014/11/21 16:52:46 [warning] [application] Failed to set unsafe attribute "req_file1" of "PermitRequirements".
in E:\xampp\htdocs\EngModule\protected\controllers\PermitRequirementsController.php (80)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/21 18:19:16 [error] [exception.CException] exception 'CException' with message 'Property "ElectricalInstallationsPermit.remarks" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('remarks')
#1 E:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('remarks')
#2 E:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(ElectricalInstallationsPermit), 'remarks')
#3 E:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(ElectricalInstallationsPermit), 'remarks', Array)
#4 E:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(ElectricalInstallationsPermit), 'remarks', Array)
#5 E:\xampp\htdocs\EngModule\protected\views\electricalInstallationspermit\_approve_permit.php(72): CActiveForm->textField(Object(ElectricalInstallationsPermit), 'remarks', Array)
#6 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#7 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', Array, true)
#8 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', Array, true)
#9 E:\xampp\htdocs\EngModule\protected\views\electricalInstallationspermit\approve_permit.php(19): CController->renderPartial('_approve_permit', Array)
#10 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#11 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', Array, true)
#12 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', Array, true)
#13 E:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('approve_permit', Array, true)
#14 E:\xampp\htdocs\EngModule\protected\controllers\ElectricalInstallationsPermitController.php(237): CController->render('approve_permit', Array)
#15 [internal function]: ElectricalInstallationsPermitController->actionApprove_permit('1541896FF6B546D...', NULL)
#16 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ElectricalInstallationsPermitController), Array)
#17 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ElectricalInstallationsPermitController), Object(ReflectionMethod), Array)
#18 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#20 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#22 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#23 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#24 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#25 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('approve_permit')
#27 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('electricalInsta...')
#28 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#29 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#30 {main}
REQUEST_URI=/EngModule/index.php?r=electricalInstallationsPermit/approve_permit&id=1541896FF6B546D33414A56669B89029
HTTP_REFERER=http://localhost/EngModule/index.php?r=electricalInstallationsPermit/view&permit_num=16
---
2014/11/21 18:20:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('reject_permit')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('electricalInsta...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=electricalInstallationsPermit/reject_permit&id=1541896FF6B546D33414A56669B89029
HTTP_REFERER=http://localhost/EngModule/index.php?r=electricalInstallationsPermit/view&permit_num=16
---
2014/11/21 18:23:13 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('reject_permit')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('electricalInsta...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=electricalInstallationsPermit/reject_permit&id=DDC199B87F0F5C6EF4F70A63D9F2CC99
HTTP_REFERER=http://localhost/EngModule/index.php?r=electricalInstallationsPermit/view&permit_num=20
---
2014/11/21 18:26:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\ElectricalInstallationsPermitController.php:303
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\ElectricalInstallationsPermitController.php(248): ElectricalInstallationsPermitController->loadModelPending('AA22211A0A8A5CB...', NULL)
#1 [internal function]: ElectricalInstallationsPermitController->actionReject_permit('AA22211A0A8A5CB...', NULL)
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ElectricalInstallationsPermitController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ElectricalInstallationsPermitController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('reject_permit')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('electricalInsta...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=electricalInstallationsPermit/reject_permit&id=AA22211A0A8A5CB15C0F5C9432754CBD
HTTP_REFERER=http://localhost/EngModule/index.php?r=electricalInstallationsPermit/view&permit_num=21
---
2014/11/22 02:37:27 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 02:40:26 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 02:54:23 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 02:56:02 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:01:21 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (151)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:04:05 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (151)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:04:36 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (151)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:06:40 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:19:27 [warning] [application] Failed to set unsafe attribute "req_file1" of "PermitRequirements".
in E:\xampp\htdocs\EngModule\protected\controllers\PermitRequirementsController.php (80)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:24:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php:381
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(197): BuildingPermitController->loadModelPending('A5C31576FD6E039...', NULL)
#1 [internal function]: BuildingPermitController->actionUpdate('A5C31576FD6E039...', NULL)
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/update&id=A5C31576FD6E039BD551EC8788BB3F29
HTTP_REFERER=http://localhost/EngModule/index.php?r=permitrequirements/view&permit_num=86
---
2014/11/22 03:24:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php:360
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(84): BuildingPermitController->loadModel('86')
#1 [internal function]: BuildingPermitController->actionView('86', NULL)
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/view&id=86
HTTP_REFERER=http://localhost/EngModule/index.php?r=permitrequirements/update&id=A5C31576FD6E039BD551EC8788BB3F29
---
2014/11/22 03:25:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php:360
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(84): BuildingPermitController->loadModel('86')
#1 [internal function]: BuildingPermitController->actionView('86', NULL)
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/view&id=86
HTTP_REFERER=http://localhost/EngModule/index.php?r=permitrequirements/update&id=A5C31576FD6E039BD551EC8788BB3F29
---
2014/11/22 03:28:58 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/delete&id=F11466E45E48EF17EB8E08418358F38F
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/view&id=F11466E45E48EF17EB8E08418358F38F
---
2014/11/22 03:30:12 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/delete&id=F11466E45E48EF17EB8E08418358F38F
---
2014/11/22 03:48:37 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 03:50:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/admin
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/view&permit_num=85
---
2014/11/22 04:04:03 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('electricalInsta...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=electricalInstallationsPermit/admin
HTTP_REFERER=http://localhost/EngModule/index.php?r=electricalInstallationsPermit/view&id=B759F6D28ECF3FEA82A3D5755086A361
---
2014/11/22 04:17:56 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 04:41:05 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/22 04:41:11 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 14:00:04 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 14:17:47 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 14:50:51 [error] [exception.CException] exception 'CException' with message 'Property "BuildingPermit.fname" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('fname')
#1 E:\xampp\htdocs\EngModule\protected\views\buildingpermit\buildingpermit.php(116): CActiveRecord->__get('fname')
#2 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#3 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', Array, true)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', Array, true)
#5 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(401): CController->renderPartial('buildingpermit', Array, true)
#6 [internal function]: BuildingPermitController->actionGeneratepdf('ECDEBE961A6D4A2...')
#7 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#8 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#9 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#16 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('generatepdf')
#18 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#19 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/generatepdf&var=ECDEBE961A6D4A2BB6C2ECD6DB61B721
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/view&permit_num=86
---
2014/11/23 14:57:46 [error] [exception.CException] exception 'CException' with message 'Property "BuildingPermit.approved_dt" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('approved_dt')
#1 E:\xampp\htdocs\EngModule\protected\views\buildingpermit\buildingpermit.php(120): CActiveRecord->__get('approved_dt')
#2 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#3 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', Array, true)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', Array, true)
#5 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(403): CController->renderPartial('buildingpermit', Array, true)
#6 [internal function]: BuildingPermitController->actionGeneratepdf('ECDEBE961A6D4A2...')
#7 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#8 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#9 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#16 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('generatepdf')
#18 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#19 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/generatepdf&var=ECDEBE961A6D4A2BB6C2ECD6DB61B721
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/view&permit_num=86
---
2014/11/23 15:06:16 [warning] [application] Failed to set unsafe attribute "owner_tin" of "BuildingPermit".
in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php (110)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:06:55 [warning] [application] Failed to set unsafe attribute "req_file1" of "PermitRequirements".
in E:\xampp\htdocs\EngModule\protected\controllers\PermitRequirementsController.php (80)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:07:33 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:10:20 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:10:42 [warning] [application] Failed to set unsafe attribute "owner_tin" of "BuildingPermit".
in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php (110)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:11:02 [warning] [application] Failed to set unsafe attribute "req_file1" of "PermitRequirements".
in E:\xampp\htdocs\EngModule\protected\controllers\PermitRequirementsController.php (80)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:12:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\PermitRequirementsController.php:291
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\PermitRequirementsController.php(57): PermitRequirementsController->loadModelByPermitNum('89')
#1 [internal function]: PermitRequirementsController->actionView(NULL, '89')
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(PermitRequirementsController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(PermitRequirementsController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('permitrequireme...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=permitrequirements/view&permit_num=89
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/view&permit_num=89
---
2014/11/23 15:25:05 [error] [php] include(CitizenRegisterForm.php): failed to open stream: No such file or directory (E:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(115): spl_autoload_call()
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CitizenUserController->runAction()
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CitizenUserController->filterAccessControl()
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CitizenUserController->runActionWithFilters()
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CitizenUserController->run()
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CWebApplication->run()
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php (115)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:25:16 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "unique_username"
DETAIL:  Key (username)=(testuser1) already exists.. The SQL statement executed was: INSERT INTO "tbl_citizen_user" ("username", "password", "firstName", "middleName", "lastName", "addr_num", "addr_street", "addr_brgy", "addr_city_mun", "tin", "employment_info", "education", "email_add", "mobile_num", "app_status", "create_dt", "id") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, NOW(), :yp15).
in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php (133)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:25:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "unique_username"
DETAIL:  Key (username)=(testuser1) already exists.. The SQL statement executed was: INSERT INTO "tbl_citizen_user" ("username", "password", "firstName", "middleName", "lastName", "addr_num", "addr_street", "addr_brgy", "addr_city_mun", "tin", "employment_info", "education", "email_add", "mobile_num", "app_status", "create_dt", "id") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, NOW(), :yp15)' in E:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(133): CActiveRecord->save()
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:27:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('23423421')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:29:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:328
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('123131')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:30:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:319
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('12312')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:34:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:319
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('7654')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:37:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:320
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('5431')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:40:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:320
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('1231')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:40:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('321')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:41:25 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 15:41:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:322
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('12312')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:48:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:319
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(121): CitizenUserController->loadModelByTIN('12312')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:48:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:304
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(120): CitizenUserController->loadModelByUsername('qw')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:51:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:300
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php(120): CitizenUserController->loadModelByUsername('as')
#1 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CitizenUserController->actionCreate()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/cre...')
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:53:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "duplicatedetected".' in E:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('duplicatedetect...')
#1 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('duplicatedetect...')
#2 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/duplicated...')
#3 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/EngModule/index.php?r=site/duplicatedetected
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 15:54:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "error_page".' in E:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('error_page')
#1 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('error_page')
#2 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/error_page')
#3 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/EngModule/index.php?r=site/error_page&message=Username+already+exist.
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/create
---
2014/11/23 16:12:18 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 16:14:01 [error] [php] include(ermitAppStatusEnum.php): failed to open stream: No such file or directory (E:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\views\site\index.php(10): spl_autoload_call()
#1 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#2 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#3 E:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#4 E:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#5 E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php(32): SiteController->render()
#6 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 E:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CWebApplication->run()
REQUEST_URI=/EngModule/index.php?r=site/index
in E:\xampp\htdocs\EngModule\protected\views\site\index.php (10)
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (32)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 16:15:56 [error] [exception.CException] exception 'CException' with message 'BuildingPermit and its behaviors do not have a method or closure named "findAllStatus".' in E:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(227): CComponent->__call('findAllStatus', Array)
#1 E:\xampp\htdocs\EngModule\protected\views\site\index.php(10): CActiveRecord->__call('findAllStatus', Array)
#2 E:\xampp\htdocs\EngModule\protected\views\site\index.php(10): BuildingPermit->findAllStatus(0)
#3 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#4 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', NULL, true)
#5 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', NULL, true)
#6 E:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#7 E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php(32): CController->render('index')
#8 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=site/index
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit
---
2014/11/23 16:16:24 [error] [php] htmlspecialchars() expects parameter 1 to be string, object given (E:\xampp\htdocs\yii\framework\web\helpers\CHtml.php:103)
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#1 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#3 E:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#4 E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php(32): SiteController->render()
#5 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#8 E:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#9 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 E:\xampp\htdocs\EngModule\index.php(13): CWebApplication->run()
REQUEST_URI=/EngModule/index.php?r=site/index
in E:\xampp\htdocs\EngModule\protected\views\site\index.php (10)
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (32)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 16:24:46 [error] [exception.CException] exception 'CException' with message 'Property "BuildingPermit.tableName" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('tableName')
#1 E:\xampp\htdocs\EngModule\protected\models\BuildingPermit.php(201): CActiveRecord->__get('tableName')
#2 E:\xampp\htdocs\EngModule\protected\views\site\index.php(10): BuildingPermit->countAllByStatus(0)
#3 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('E:\\xampp\\htdocs...')
#4 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('E:\\xampp\\htdocs...', NULL, true)
#5 E:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('E:\\xampp\\htdocs...', NULL, true)
#6 E:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#7 E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php(32): CController->render('index')
#8 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 E:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=site/index
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit
---
2014/11/23 16:29:14 [error] [php] include(ElectricalInstallationPermit.php): failed to open stream: No such file or directory (E:\xampp\htdocs\yii\framework\YiiBase.php:427)
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\views\site\index.php(13): spl_autoload_call()
#1 E:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#2 E:\xampp\htdocs\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#3 E:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#4 E:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#5 E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php(32): SiteController->render()
#6 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 E:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 E:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 E:\xampp\htdocs\EngModule\index.php(13): CWebApplication->run()
REQUEST_URI=/EngModule/index.php?r=site/index
in E:\xampp\htdocs\EngModule\protected\views\site\index.php (13)
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (32)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/23 16:30:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "electricalinstallationpermit/index".' in E:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('electricalinsta...')
#1 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/EngModule/index.php?r=electricalinstallationpermit/index
HTTP_REFERER=http://localhost/EngModule/index.php?r=site/index
---
2014/11/24 14:17:08 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:38:29 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:39:30 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:39:30 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "unique_emp_tin"
DETAIL:  Key (tin)=(12312) already exists.. The SQL statement executed was: INSERT INTO "tbl_lgu_user" ("user_type", "username", "password", "firstName", "middleName", "lastName", "employee_no", "position", "tin", "civil_status", "gender", "id") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11).
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (123)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:39:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "unique_emp_tin"
DETAIL:  Key (tin)=(12312) already exists.. The SQL statement executed was: INSERT INTO "tbl_lgu_user" ("user_type", "username", "password", "firstName", "middleName", "lastName", "employee_no", "position", "tin", "civil_status", "gender", "id") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11)' in E:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php(123): CActiveRecord->save()
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LguUserController->actionCreate()
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('lguUser/create')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=lguUser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=lguUser/create
---
2014/11/24 15:42:42 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:43:30 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:44:45 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:46:08 [warning] [application] Failed to set unsafe attribute "userType_temp" of "LGU_User".
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (100)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:53:35 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "unique_emp_tin"
DETAIL:  Key (tin)=(12312) already exists.. The SQL statement executed was: INSERT INTO "tbl_lgu_user" ("user_type", "username", "password", "firstName", "middleName", "lastName", "employee_no", "position", "tin", "civil_status", "gender", "id") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11).
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (124)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:53:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "unique_emp_tin"
DETAIL:  Key (tin)=(12312) already exists.. The SQL statement executed was: INSERT INTO "tbl_lgu_user" ("user_type", "username", "password", "firstName", "middleName", "lastName", "employee_no", "position", "tin", "civil_status", "gender", "id") VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11)' in E:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php(124): CActiveRecord->save()
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LguUserController->actionCreate()
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('lguUser/create')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=lguUser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=lguUser/create
---
2014/11/24 15:55:05 [error] [php] Undefined variable: username (E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php:103)
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): LguUserController->runAction()
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): LguUserController->filterAccessControl()
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): LguUserController->runActionWithFilters()
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): LguUserController->run()
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CWebApplication->run()
REQUEST_URI=/EngModule/index.php?r=lguUser/create
in E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php (103)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:58:30 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 15:59:51 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 16:32:58 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 16:45:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:101
Stack trace:
#0 [internal function]: CitizenUserController->actionView(NULL, '')
#1 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(CitizenUserController), Array)
#2 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(CitizenUserController), Object(ReflectionMethod), Array)
#3 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/vie...')
#13 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/view&username=
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/change_password&username=testuser1
---
2014/11/24 16:47:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\CitizenUserController.php:101
Stack trace:
#0 [internal function]: CitizenUserController->actionView(NULL, '')
#1 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(CitizenUserController), Array)
#2 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(CitizenUserController), Object(ReflectionMethod), Array)
#3 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('citizenuser/vie...')
#13 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/EngModule/index.php?r=citizenuser/view&username=
HTTP_REFERER=http://localhost/EngModule/index.php?r=citizenuser/change_password&username=testuser1
---
2014/11/24 16:50:38 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 16:57:03 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 16:57:18 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 E:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(WebUser), 'You are not aut...')
#1 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('change_password')
#8 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('lguUser/change_...')
#9 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/EngModule/index.php?r=lguUser/change_password&username=admin1
HTTP_REFERER=http://localhost/EngModule/index.php?r=lguUser/view&id=67F89B6F8946CDFF3CE9E6737D9E6C97
---
2014/11/24 17:00:26 [error] [exception.CException] exception 'CException' with message 'Property "LGU_User.update_dt" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('update_dt', Object(CDbExpression))
#1 E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php(255): CActiveRecord->__set('update_dt', Object(CDbExpression))
#2 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LguUserController->actionChange_password()
#3 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('change_password')
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('lguUser/change_...')
#13 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/EngModule/index.php?r=lguUser/change_password&username=admin1
HTTP_REFERER=http://localhost/EngModule/index.php?r=lguUser/change_password&username=admin1
---
2014/11/24 17:00:56 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 17:01:05 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 17:07:14 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/24 17:07:49 [error] [exception.CException] exception 'CException' with message 'Property "LGU_User.created_by" is not defined.' in E:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('created_by', 'admin1')
#1 E:\xampp\htdocs\EngModule\protected\controllers\LguUserController.php(124): CActiveRecord->__set('created_by', 'admin1')
#2 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LguUserController->actionCreate()
#3 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('lguUser/create')
#13 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/EngModule/index.php?r=lguUser/create
HTTP_REFERER=http://localhost/EngModule/index.php?r=lguUser/create
---
2014/11/25 13:49:43 [warning] [application] Failed to set unsafe attribute "rememberMe" of "CitizenLoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (125)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/25 14:13:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php:384
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(196): BuildingPermitController->loadModelPending(NULL, '85')
#1 [internal function]: BuildingPermitController->actionUpdate(NULL, '85')
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/update&permit_num=85
---
2014/11/25 14:14:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php:384
Stack trace:
#0 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(196): BuildingPermitController->loadModelPending(NULL, '85')
#1 [internal function]: BuildingPermitController->actionUpdate(NULL, '85')
#2 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#3 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#4 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#13 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#14 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/update&permit_num=85
---
2014/11/25 14:18:59 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42883]: Undefined function: 7 ERROR:  function utc_timestamp() does not exist
LINE 1: ...rans_cap"=$26, "total_gen_cap"=$27, "update_date"=UTC_TIMEST...
                                                             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.. The SQL statement executed was: UPDATE "tbl_building_permit" SET "id"=:yp0, "owner_fname"=:yp1, "request_date"=:yp2, "owner_phone"=:yp3, "owner_mname"=:yp4, "owner_lname"=:yp5, "econ_act"=:yp6, "ownership_form"=:yp7, "work_scope"=:yp8, "tax_dec_num"=:yp9, "occupation_type"=:yp10, "addr_num"=:yp11, "addr_street"=:yp12, "addr_brgy"=:yp13, "addr_city_mun"=:yp14, "bldg_lot_num"=:yp15, "bldg_block_num"=:yp16, "bldg_tct_num"=:yp17, "const_street"=:yp18, "const_brgy"=:yp19, "const_city_mun"=:yp20, "permit_num"=:yp21, "app_num"=:yp22, "app_status"=:yp23, "total_const_load"=:yp24, "total_trans_cap"=:yp25, "total_gen_cap"=:yp26, "update_date"=UTC_TIMESTAMP(), "updated_by"=:yp27, "approved_by"=:yp28, "approved_by_position"=:yp29, "requested_by"=:yp30, "approval_date"=:yp31, "owner_tin"=:yp32, "remarks"=:yp33 WHERE "tbl_building_permit"."id"='07BFF0DBDA6DC0F7E606205223FC7524'.
in E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php (208)
in E:\xampp\htdocs\EngModule\index.php (13)
2014/11/25 14:18:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42883]: Undefined function: 7 ERROR:  function utc_timestamp() does not exist
LINE 1: ...rans_cap"=$26, "total_gen_cap"=$27, "update_date"=UTC_TIMEST...
                                                             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.. The SQL statement executed was: UPDATE "tbl_building_permit" SET "id"=:yp0, "owner_fname"=:yp1, "request_date"=:yp2, "owner_phone"=:yp3, "owner_mname"=:yp4, "owner_lname"=:yp5, "econ_act"=:yp6, "ownership_form"=:yp7, "work_scope"=:yp8, "tax_dec_num"=:yp9, "occupation_type"=:yp10, "addr_num"=:yp11, "addr_street"=:yp12, "addr_brgy"=:yp13, "addr_city_mun"=:yp14, "bldg_lot_num"=:yp15, "bldg_block_num"=:yp16, "bldg_tct_num"=:yp17, "const_street"=:yp18, "const_brgy"=:yp19, "const_city_mun"=:yp20, "permit_num"=:yp21, "app_num"=:yp22, "app_status"=:yp23, "total_const_load"=:yp24, "total_trans_cap"=:yp25, "total_gen_cap"=:yp26, "update_date"=UTC_TIMESTAMP(), "updated_by"=:yp27, "approved_by"=:yp28, "approved_by_position"=:yp29, "requested_by"=:yp30, "approval_date"=:yp31, "owner_tin"=:yp32, "remarks"=:yp33 WHERE "tbl_building_permit"."id"='07BFF0DBDA6DC0F7E606205223FC7524'' in E:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1748): CDbCommand->execute()
#1 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1124): CActiveRecord->updateByPk('07BFF0DBDA6DC0F...', Array)
#2 E:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->update(NULL)
#3 E:\xampp\htdocs\EngModule\protected\controllers\BuildingPermitController.php(208): CActiveRecord->save()
#4 [internal function]: BuildingPermitController->actionUpdate(NULL, '89')
#5 E:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BuildingPermitController), Array)
#6 E:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BuildingPermitController), Object(ReflectionMethod), Array)
#7 E:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 E:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 E:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 E:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 E:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 E:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#14 E:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#16 E:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('buildingpermit/...')
#17 E:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 E:\xampp\htdocs\EngModule\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/EngModule/index.php?r=buildingpermit/update&permit_num=89
HTTP_REFERER=http://localhost/EngModule/index.php?r=buildingpermit/update&permit_num=89
---
2014/11/25 14:45:38 [warning] [application] Failed to set unsafe attribute "rememberMe" of "LoginForm".
in E:\xampp\htdocs\EngModule\protected\controllers\SiteController.php (94)
in E:\xampp\htdocs\EngModule\index.php (13)
